### PR TITLE
Pr17

### PR DESCRIPTION
**Pull request by wraithgar**
Updated version of #17 that uses _indexedGet instead of indexOf

Tests still pass.  Pinging @herkyl to review.

### CODE REVIEW

File: ampersand-filtered-subcollection.js [hunk 1]
```diff
@@ -322,6 +322,9 @@ assign(FilteredCollection.prototype, Events, {
             if (!this._testModel(model) || alreadyHave) {
                 action = 'ignore';
             }
+        } else if (eventName === 'change' && !this._indexedGet(model)) {
```
Comment (legastero) [inline ampersand-filtered-subcollection.js]: Use the existing `alreadyHave` variable?

**Comment by kamilogorek** [comment]
:+1:

**Comment by wraithgar** [comment]
Thank you again @herkyl for getting this 99% done and providing solid tests around it.

**Comment by wraithgar** [comment]
released as `v2.0.2`

**Comment by herkyl** [comment]
Thanks @wraithgar for accepting it!

**Comment by herkyl** [comment]
Btw #17 and #16 didn't get closed automatically

**Pull request merged**